CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+# SPDX-FileCopyrightText: 2025 Klarälvdalens Datakonsult AB, a KDAB Group
+# company <info@kdab.com>
+#
+# SPDX-License-Identifier: GPL-2.0-only OR GPL-3.0-only
+
+cmake_minimum_required(VERSION 3.21)
+project(declarativewidgets)
+
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_INCLUDE_CURRENT_DIRS ON)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+
+option(USE_QT6 "Use Qt6" OFF)
+option(BUILD_EXAMPLES "Build the examples" ON)
+option(ENABLE_SANITIZERS "Enable asan/ubsan sanitizers" OFF)
+
+if(USE_QT6)
+  find_package(Qt6 6.5 NO_MODULE REQUIRED COMPONENTS Qml Widgets QuickWidgets)
```

Comment:
just to be sure: 6.5 is the minimum version, not the exact version, right?

Reply:
yes, it's minimum. It's arbitrary though, 6.2 might work, or not.
Not worth to find out I guess.